### PR TITLE
Read functionality

### PR DESCRIPTION
**Pull request by Bpompa95**

### CODE REVIEW

**Comment by Bpompa95** [comment]
working progress just want you guys to review

**Review by iameddieyayaya — COMMENTED** [review]
so far so good

File: app/javascript/components/App.js [hunk 1]
```diff
@@ -74,10 +76,17 @@ class App extends React.Component {
             path="/fatenew"
             render={(props) => <FateNew createNewEvent={this.createNewEvent} />}
           />
+          <Route path="/fateshow/:id" render={(props) => { 
+            let id = props.match.params.id
+            let event = this.state.events.find(event => event.id === +id)
+            {console.log(this.state.events);}
```
Comment (iameddieyayaya) [inline app/javascript/components/App.js:82]: do you need this log?

File: app/javascript/components/pages/FateShow.js [hunk 2]
```diff
@@ -1,13 +1,34 @@
...
+                        </div>
+                    )}
+                </div>
+            </>
 
         )
     }
 }
 export default FateShow
+{/* {this.props.event && this.props.event.map((event, i) => {
```
Comment (iameddieyayaya) [inline app/javascript/components/pages/FateShow.js:25]: not using it, delete it?

**Review by iameddieyayaya — APPROVED** [review]
Can be cleaned up a bit, looks fine.

**Review by WalkerTexas-coder — APPROVED** [review]
Looks good!

**Pull request merged**